app/models/editionable_worldwide_organisation.rb

```diff
@@ -4,4 +4,20 @@ class EditionableWorldwideOrganisation < Edition
   def display_type_key
     "editionable_worldwide_organisation"
   end
+
+  def publishing_api_presenter
+    PublishingApi::EditionableWorldwideOrganisationPresenter
+  end
+
+  def base_path
+    "/editionable-world/organisations/#{slug}"
```

Comment:
I assume we'll have to change this before these are made live to publishers?

Reply:
yeah, we'll also have to republish everything i think, but it'll give us a path in integration to check stuff out. We'll probably need to make some changes to government-frontend to render stuff at this path

Reply:
I _think_ government-frontend uses the schema to decide how to render a page, rather than the route, so hopefully no changes needed there 🤞.